dockstore-integration-testing/src/test/java/io/dockstore/webservice/TokenResourceIT.java

```diff
@@ -405,17 +367,15 @@ private void addUnavailableGoogleTokenToGitHubUser() {
      * Dockstore account 1: has GOOGLE_ACCOUNT_USERNAME2 Google account linked
      * Dockstore account 2: has GITHUB_ACCOUNT_USERNAME GitHub account linked
      * Trying to link GITHUB_ACCOUNT_USERNAME GitHub account to Dockstore account 1 should fail
-     * @throws Exception
      */
-    private void addUnavailableGitHubTokenToGoogleUser() throws Exception {
-        mockGitHub(CUSTOM_USERNAME1);
+    private void addUnavailableGitHubTokenToGoogleUser() {
         TokensApi otherUserTokensApi = new TokensApi(getWebClient(true, GOOGLE_ACCOUNT_USERNAME2));
         try {
-            otherUserTokensApi.addGithubToken("potato");
+            otherUserTokensApi.addGithubToken(getFakeCode(SUFFIX1));
             Assert.fail();
         } catch (ApiException e){
             Assert.assertEquals(HttpStatus.SC_INTERNAL_SERVER_ERROR, e.getCode());
-            Assert.assertTrue(e.getMessage().contains("already exists"));;
```

Comment:
Amusing, how did you notice this?

Reply:
FindBugs in the IDE.  There a green checkmark when everything is ok.